src/versionbits.h

```diff
@@ -30,6 +30,15 @@ enum ThresholdState {
 // will either be NULL or a block with (height + 1) % Period() == 0.
 typedef std::map<const CBlockIndex*, ThresholdState> ThresholdConditionCache;
 
+struct BIP9DeploymentInfo {
```

Comment:
I mean, I would say the only reason for not having versionbits in the consensus package already is because it still depends on chain.o (which is storage-dependent).